src/app/qgsmeasuredialog.cpp

```diff
@@ -43,7 +43,20 @@ QgsMeasureDialog::QgsMeasureDialog( QgsMeasureTool* tool, Qt::WindowFlags f )
   mMeasureArea = tool->measureArea();
   mTotal = 0.;
 
+  mUnitsCombo->addItem( tr( "Meters" ) );
+  mUnitsCombo->addItem( tr( "Feet" ) );
+  mUnitsCombo->addItem( tr( "Degrees" ) );
+  mUnitsCombo->addItem( tr( "Nautical Miles" ) );
+
+  QSettings settings;
+  QString units = settings.value( "/qgis/measure/displayunits", "meters" ).toString();
+  mUnitsCombo->setCurrentIndex( mUnitsCombo->findText( units, Qt::MatchFixedString ) );
+
   updateSettings();
+
+  connect( mUnitsCombo, SIGNAL( currentIndexChanged( const QString & ) ), this, SLOT( unitsChanged( const QString & ) ) );
+
+  mTable->headerItem()->setTextAlignment( 0, Qt::AlignRight );
```

Comment:
Not sure you can. Happy to move it there if we can.

On Fri, 2 Jan 2015 11:54 pm Salvatore Larosa <notifications@github.com>
wrote:

> In src/app/qgsmeasuredialog.cpp
> <https://github.com/qgis/QGIS/pull/1747#discussion-diff-22410601>:
>
> >    updateSettings();
> > +
> > +  connect( mUnitsCombo, SIGNAL( currentIndexChanged( const QString & ) ), this, SLOT( unitsChanged( const QString & ) ) );
> > +
> > +  mTable->headerItem()->setTextAlignment( 0, Qt::AlignRight );
>
> @NathanW2 <https://github.com/NathanW2> not a big deal, I think it's
> better to define this within UI file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qgis/QGIS/pull/1747/files#r22410601>.
>